src/sentry/snuba/metrics/extraction.py

```diff
@@ -49,6 +52,67 @@
 
 logger = logging.getLogger(__name__)
 
+
+# This helps us control the different spec versions
+# in order to migrate customers from invalid specs
+class SpecVersion(NamedTuple):
+    version: int
```

Comment:
The version may not make sense anymore since we do not use `MIN_VERSION` or querying for a spec by the index.